openwebstart/src/main/java/com/openwebstart/jvm/ui/dialogs/JVMServerWhitelistDialog.java

```diff
@@ -0,0 +1,47 @@
+package com.openwebstart.jvm.ui.dialogs;
+
+import com.openwebstart.controlpanel.ButtonPanelFactory;
+import com.openwebstart.ui.ModalDialog;
+import net.adoptopenjdk.icedteaweb.client.controlpanel.panels.AbstractServerWhitelistPanel;
+import net.adoptopenjdk.icedteaweb.client.util.UiLock;
+import net.adoptopenjdk.icedteaweb.i18n.Translator;
+import net.sourceforge.jnlp.config.DeploymentConfiguration;
+import net.sourceforge.jnlp.util.whitelist.UrlWhiteListUtils;
+import net.sourceforge.jnlp.util.whitelist.WhitelistEntry;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import java.awt.BorderLayout;
+import java.util.List;
+
+import static com.openwebstart.config.OwsDefaultsProvider.JVM_SERVER_WHITELIST;
+
+public class JVMServerWhitelistDialog extends ModalDialog {
+
+    //private static final Logger LOG = LoggerFactory.getLogger(JVMServerWhitelistDialog.class);
+
+    private final Translator translator = Translator.getInstance();
+    private final JButton okButton;
+
+    public JVMServerWhitelistDialog(final DeploymentConfiguration deploymentConfiguration) {
+        setTitle(translator.translate("dialog.jvmManagerConfig.title"));
+
+        final UiLock uiLock = new UiLock(deploymentConfiguration);
+        okButton = new JButton(translator.translate("action.ok"));
```

Comment:
does ok make sense? I would like to have `close` as an option and not ok. What do you think?